src/core/api/link_raw_api.cpp

```diff
@@ -67,6 +84,9 @@ otError otLinkRawSetPanId(otInstance *aInstance, uint16_t aPanId)
     VerifyOrExit(aInstance->mLinkRaw.IsEnabled(), error = OT_ERROR_INVALID_STATE);
 
     otPlatRadioSetPanId(aInstance, aPanId);
```

Comment:
Yes, I see the discrepancy now.  The vision was always to have a common API for operating in "radio-only mode" regardless of whether or not OpenThread core was compiled in or out.  Do you think we can work towards that vision?